packages/api/endpoints/providers.js

```diff
@@ -15,10 +13,11 @@ const RecordDoesNotExist = require('../lib/errors').RecordDoesNotExist;
  * @return {undefined}
  */
 function list(event, cb) {
-  const search = new Search(event, 'provider');
-  search.query().then(response => cb(null, response)).catch((e) => {
-    cb(e);
-  });
+  const providers = new models.Provider();
+  return providers
+    .scan()
```

Comment:
We should continue to use the ElasticSearch for returning results for the list endpoint, otherwise, none of our query parameters will work.